src/styles/index.scss

```diff
@@ -1,3 +1,72 @@
+@import '../utils/variables';
+
+@keyframes hoverTransform {
+  from {
+    transform: translateY(0);
+  }
+
+  to {
+    transform: translateY(-20px);
+  }
```

Comment:
The `hoverTransform` keyframes are not used correctly. According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), keyframes should be used for complex animations. Here, the animation only changes the `transform` property, which can be transitioned without keyframes. Consider using the `transition` property instead.